### PR TITLE
added user model

### LINKED ISSUE #1: Create a User Model
You need to create a user model. Set up a model named `user` which stores the following fields -
- name
- username (IITK)
- roll number
- batch (eg. Y18, Y19, etc)
- program (eg. BT, BT-BS, BS, BS-BS, MT, Ph.D., etc)
- department
- GitHub profile link
- Linkedin profile link
- Master CV drive link
- Resume 1 drive link
- Resume 2 drive link

### PR DESCRIPTION
**Pull request by navneet-28**
Added the user model. The resume2 link is set to be optional.
Closes #1

### CODE REVIEW

**Comment by navneet-28** [comment]
Updated the timezone and increased the max length for batch field.

File: users/admin.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+
+from .models import User
+
+# Register your models here.
+admin.site.register(User)
```
Comment (SomTambe) [inline users/admin.py]: Add a newline at the end of the file. I don't ever remember registering my models, I don't know if this is necessary.

File: users/models.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Create your models here.
+class User(models.Model):
+    name=models.CharField(max_length=120)
+    username=models.CharField(max_length=40, unique=True)
+    batch=models.CharField(max_length=15)
+    program=models.CharField(max_length=20)
+    department=models.CharField(max_length=50)
+    githubLink=models.URLField(max_length=100,unique=True,null=True)
+    linkedinLink=models.URLField(max_length=200,unique=True,null=True)
+    mastercv=models.URLField(max_length=300,unique=True)
```
Comment (SomTambe) [inline users/models.py:12]: Make all the URL Fields of max length = `300`.

**Review by SomTambe — CHANGES_REQUESTED** [review]
Make the requested changes.

**Comment by navneet-28** [comment]
Made all the requested changes.

**Pull request closed without merging**